automation/taskcluster/decision_task.py

```diff
@@ -263,12 +328,39 @@ def create_compare_locales_task():
         sys.exit(2)
 
     print("Got variants: " + ' '.join(variants))
+    queue = taskcluster.Queue({'baseUrl': 'http://taskcluster/queue/v1'})
+    task_graph = TaskGraph(queue)
 
+    arm_assemble_task_id = None
+    aarch64_assemble_task_id = None
     for variant in variants:
-        lib.tasks.schedule_task(queue, taskcluster.slugId(), create_variant_assemble_task(variant))
-        lib.tasks.schedule_task(queue, taskcluster.slugId(), create_variant_test_task(variant))
+        task_graph.schedule_new_task(create_variant_test_task(variant))
+        assemble_task_id = task_graph.schedule_new_task(create_variant_assemble_task(variant))
+
+        arch, build_type = _get_architecture_and_build_type_from_variant(variant)
+        if build_type == 'debug' and arch == 'arm':
+            arm_assemble_task_id = assemble_task_id
+        elif build_type == 'debug' and arch == 'aarch64':
+            aarch64_assemble_task_id = assemble_task_id
+
+    if command == 'master-push':
+        populate_chain_of_trust_required_but_unused_files()
```

Comment:
Nit: this call isn't needed since we remake the file later in that function https://github.com/mozilla-mobile/reference-browser/pull/681/files#diff-168d551d2c0830884a5caeb50ae576e9R366